swabbie-core/src/main/kotlin/com/netflix/spinnaker/swabbie/model/Resource.kt

```diff
@@ -69,7 +69,7 @@ abstract class Resource : Excludable, Timestamped, HasDetails() {
       return true
     }
     return other is Resource &&
-      other.resourceId == this.resourceId && other.resourceType == this.resourceId
+      other.resourceId == this.resourceId && other.resourceType == this.resourceType
```

Comment:
lol

Reply:
This is what got me thinking about the equals/hashcode. Obviously a bug otherwise all equality checks would fail.
Instead the default `equals` on the data class would be called. Explains why we haven't run into trouble.